### PR TITLE
Exemplified menu entries for change case menu

### LINKED ISSUE #9339: Convert Case: exemplify the menus
**Is your suggestion for improvement related to a problem? Please describe.**
Just trying to make the UI more practical.

Currently, the sub-menus of `Change case` looks like this:

ChangeCase

If you have no background in typography, the meanings are not obvious (Capitalize vs Upper case. Sentence case vs Title case)

**Describe the solution you'd like**
Alter the sub-menus into:
 ```
Capitalize
 lower case
 Sentence case
 Title Case
 UPPER CASE
```

And alter the tips (displayed when you stay long over the menu) accordingly. Example for Title Case:
`Capitalize all words, but converts articles, prepositions, and conjunctions to lower case.`
will become
`Capitalize all Words, but Converts Articles, Prepositions, and Conjunctions to Lower Case.`

Tip: change JabRef UI into French language to looks what it will look like (yeah, this is already implemented in French)

### PR DESCRIPTION
**Pull request by manu-vaillant-afk**
Fixes #9339

We modified the Change case sub-menus and their corresponding tips (displayed when you stay long over the menu) to properly reflect exemplified cases.

I've simply changed the text visible in the sub-menus and their tips to match exemplified cases [#9339](https://github.com/Jabref/jabref/issues/9339).

Here is a screenshot of the modified sub-menus.
image

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [/] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Siedlerchr** [comment]
Have a look at the failing tests please and also the  l10n https://devdocs.jabref.org/code-howtos/localization.html

File: src/main/resources/l10n/JabRef_en.properties [hunk 1]
```diff
@@ -1469,7 +1469,7 @@ Converts\ units\ to\ LaTeX\ formatting.=Converts units to LaTeX formatting.
 HTML\ to\ LaTeX=HTML to LaTeX
 LaTeX\ cleanup=LaTeX cleanup
 LaTeX\ to\ Unicode=LaTeX to Unicode
-Lower\ case=Lower case
+Lower\ case=lower case
```
Comment (Siedlerchr) [inline src/main/resources/l10n/JabRef_en.properties:1472]: The keys on the left and the values must be equal,

**Comment by Siedlerchr** [comment]
You can ignore mac. It does not work on forks. The unit test fails because the left and right side in the properties file must be equal. This is for the english version

**Comment by manu-vaillant-afk** [comment]
Ok ty for mac test!
Yes I know that the test failed because values and entries should be equal. But then I don't know how am I supposed to do to exemplifies menu entries ? In the french version (where it is already implemented), they have not the same values/entries.

**Comment by Siedlerchr** [comment]
The test only checks the **English** version because obviously for the other languages this cannot be equal. Translators can write whatever the right translations is.

**Comment by manu-vaillant-afk** [comment]
Ok but where am I supposed to put the exemplifies menu entries string if I can't put them in **JabRef_en.properties** like this ?

**Comment by Siedlerchr** [comment]
@manu-vaillant-afk  You changed the right side (the values) of the properties. You now have to change the left side (the keys) as well in the en.properties file.

File: src/main/resources/l10n/JabRef_en.properties (hunk 1 above)
Comment (Siedlerchr) [reply src/main/resources/l10n/JabRef_en.properties:1472]:
```suggestion
lower\ case=lower case
```

**Comment by manu-vaillant-afk** [comment]
@Siedlerchr if I change the keys in **en.properties file**, I'll also have to change keys in all other languages properties files.
And also all the functions getName() with the new key in all casechanger formatter right ?
 
For example, in the file **java/org/jabref/logic/formatter/casechanger/LowerCaseFormatter.java** :
I'll turn this :
`    
    public String getName() {
        return Localization.lang("Lower case");
    }`
into this :
` public String getName() {
        return Localization.lang("lower case");
    }`

Tell me if it's what you want and I'll make these changes.

**Comment by Siedlerchr** [comment]
Yes, you have to adjust the pieces in the code and the keys in the English properties file. You don't need to change the other language files

**Comment by manu-vaillant-afk** [comment]
@Siedlerchr I've try it locally but as I was thinking, it's not working well. 
It works well in english, but not in other language because the new keys (like "UPPER CASE" which is called in Formatter.java) is not present in other languages files.

For me, there is no reason to change these keys, I just think that the test is not very useful and could be remove ?

**Comment by calixtus** [comment]
This is why this test is only checking for the english language file consistency. This test is in my eyes very useful, as it checks for the language keys consistent to the language resources, as it makes the sources more easily readable und keeps our coding style in JabRef consistent.

You don't have to change the keys in every other language file, this is done by crowdin automatically and many users, who help us translate JabRef to many different languages.

**Comment by manu-vaillant-afk** [comment]
@calixtus  @Siedlerchr Ok now the test is successful, tell me if I have to change anything else !

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**